source/our-stacks/ios/index.md

```diff
@@ -3,4 +3,67 @@ title: iOS
+ - Maintaining and setting up Apple services and applications 
+ - Consulting with UX and clients to provide best possible product
+
+## Stack
+
+### Language
+
+##### Swift
+We've started as soon as Swift 1.2 was released and since then we have migrated all our projects through Swift 2 up to Swift 3 and we make sure all our frameworks and dependencies support all versions of the language.
+
+##### Objective-C
+Objective-C is a thing of the past for us, but occasionally we do maintain projects or update applications still written in Objective-C.
+
+### IDE
+Xcode is the IDE that is widely used amongst iOS developers and we also use it at Nodes. 
+
+### Other tools
+ - [Carthage](https://github.com/Carthage/Carthage) - our weapon of choice for dependency management, simple yet powerful
+ - [Model Boiler](https://github.com/nodes-ios/ModelBoiler) - a macOS application used to generate boilerplate code for model objects using Serializable
+ - [Charles Proxy](https://www.charlesproxy.com/) - effectively serves as a MitM proxy which captures all network traffic and helps us debug network issues
+ - [MacDown](http://macdown.uranusjr.com/) - for writing READMEs and blog posts like this one
+ - [Zeplin](https://zeplin.io/) - makes it easy to transfer designs to code by providing exact measurements, colors and assets
+ - [Postman](https://www.getpostman.com/) - all our APIs are documented through Postman allowing us to easily work together
+
+### Libraries
+These are the 3rd party libraries we usually use in all our projects. If there are other services required for a project, for example a Facebook Login, we also use their related SDKs.
+
+ - [Alamofire](https://github.com/Alamofire/Alamofire) - an elegant HTTP networking library 
+ - [Kingfisher](https://github.com/onevcat/Kingfisher) - a lightweight library for downloading and caching images from the web
+ - [UrbanAirshipSDK](https://github.com/urbanairship/ios-library) - integrates with Urban Airship to register for and handle push notifications
+ - [HockeySDK](https://github.com/bitstadium/HockeySDK-iOS) - integrates with HockeyApp to collect crash reports and show application updates during testing
+ - [SnapKit](https://github.com/SnapKit/SnapKit) - a DSL to make Auto Layout easy
+ - [netfox](https://github.com/kasketis/netfox) - one line setup network debugging library useful during QA process
+ - [GoogleAnalytics](https://developers.google.com/analytics/devguides/collection/ios/v3/) - analytics and tracking library from Google
+ - [Mixpanel](https://github.com/mixpanel/mixpanel-iphone) - advanced analytics and user tracking library with features like A/B testing and push notifications
+ 
+### Open source
+iOS team at Nodes strives to deliver the best and nicest product and to allow us to do so, we've created many open source frameworks to make the development process faster, easier and safer. All our repositories are being maintained at [Github](https://github.com/nodes-ios/).
```

Comment:
* missing article in the beginning: 'The iOS team at Nodes'
* maybe change 'maintained at Github' with 'maintained on our GitHub' or 'maintained on our GitHub organisation account'. 'maintained at Github' sounds a bit weird to me.